src/library/treeitemmodel.cpp

```diff
@@ -161,6 +163,11 @@ TreeItem* TreeItemModel::setRootItem(std::unique_ptr<TreeItem> pRootItem) {
     return getRootItem();
 }
 
+/// Returns the QModelIndex of the Root element.
```

Comment:
The doc comments need to be moved into the .h file. In the .cpp file they are useless.